client/state/plans/test/selectors.js

```diff
@@ -113,4 +117,33 @@ describe( 'selectors', () => {
 			expect( price ).to.eql( null );
 		} );
 	} );
+
+	describe( '#getPlanSlug()', () => {
+		it( 'should return plan product_slug', () => {
+			const state = getStateInstance();
```

Comment:
Alright, agreed. I used fixtures because I saw them and thought that: "if we have those fixtures, why should I duplicate them?". But I see your point.